torch/fx/operator_schemas.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import inspect
+import numbers
+import typing
+import enum
+from typing import Any, Callable, Dict, List, Optional
+
+_manual_overrides : Dict[Callable, List[inspect.Signature]] = {}
+
+def _nonzero_schemas():
+    signatures = []
+
+    def nonzero(self):
+        pass
+    signatures.append(inspect.signature(nonzero))
+
+    def nonzero(self, *, as_tuple : bool):  # type: ignore
+        pass
+    signatures.append(inspect.signature(nonzero))
+
+    return signatures
+
+_manual_overrides[torch.nonzero] = _nonzero_schemas()
```

Comment:
OOC, what's special about `nonzero`? because of `as_tuple`?

Reply:
Right, `as_tuple` is handled in the Python bindings and is not present in the `aten` API: https://github.com/pytorch/pytorch/blob/957700be7e5e49e6c4eaf9e0fa351cce9fd5235b/tools/autograd/templates/python_torch_functions.cpp#L562